src/settings/Settings.ts

```diff
@@ -398,6 +398,10 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         supportedLevels: LEVELS_DEVICE_ONLY_SETTINGS,
         default: null,
     },
+    "voip_mxid_translate_pattern": {
+        supportedLevels: LEVELS_UI_FEATURE, // not a UI feature, but same level (config only, maybe .well-known in future)
```

Comment:
Hmm, reusing it for a non-UI feature seems a bit strange to me... Is there a reason you want it to participate in the settings system? For config-only things, we've typically just read it out of the config directly with `SdkConfig.get().thing`. Your docs PR also suggests its not part of "settings", but just floating in the config sea.

Reply:
oh, god point - I did originally have it as a setting but then realised exactly what you've pointed out, and failed to remove the settings entry - good spot.